modules/navier_stokes/src/auxkernels/kEpsilonViscosityAux.C

```diff
@@ -166,56 +124,83 @@ kEpsilonViscosityAux::computeValue()
     // Compute the velocity and direction of the velocity component that is parallel to the wall
     const ADReal parallel_speed = (velocity - velocity * loc_normal * loc_normal).norm();
 
-    ADReal y_plus, u_tau;
-    if (_non_equilibrium_treatment)
+    // Switch for determining the near wall quantities
+    // wall_treatment can be: "eq_newton eq_incremental eq_linearized neq"
+    ADReal u_tau;
+    ADReal y_plus;
+    ADReal mut_log; // turbulent log-layer viscosity
+    ADReal mu_wall; // total wall viscosity to obtain the shear stress at the wall
+
+    if (_wall_treatment == "eq_newton")
     {
-      // Computing non-equilibrium nondimensional wall distance and friction velocity
-      y_plus = _rho(current_argument, state) * std::pow(_C_mu, 0.25) *
-               std::pow(_k(current_argument, state), 0.5) * min_wall_dist /
-               _mu(current_argument, state);
-      const auto von_karman_value =
-          (1 / NS::von_karman_constant * std::log(NS::E_turb_constant * y_plus));
-      u_tau = std::sqrt(std::pow(_C_mu, 0.25) * std::pow(_k(current_argument, state), 0.5) *
-                        parallel_speed / von_karman_value);
+      // Full Newton-Raphson solve to find the wall quantities from the law of the wall
+      u_tau = NS::findUStar(mu, rho, parallel_speed, min_wall_dist);
```

Comment:
```suggestion
      const ADReal u_tau = NS::findUStar(mu, rho, parallel_speed, min_wall_dist);
```

Reply:
u_tau is declared above the if statements as ADReal, and then computed, so I can't declare it as const